src/pages/components/BestPostList.tsx

```diff
@@ -0,0 +1,105 @@
+import Image from 'next/image'
+import styles from '@/src/styles/BestPostList.module.css'
+import Link from 'next/link'
+import { useEffect, useState } from 'react'
+import { getDeviceType, debounce } from '@/src/utils'
+import axiosPanda from '../api/axiosPanda'
+
+const postCount = {
+  mobile: 1,
+  tablet: 2,
+  desktop: 3,
+}

```

Comment:
상수선언으로 재사용성과 관심사 분리를 잘해주셨어요!
한가지, 상수선언시에는 보통 대문자로 해주는것이 일반적입니다!
```suggestion
const POST_COUNT = {
  mobile: 1,
  tablet: 2,
  desktop: 3,
} as const;
```
대문자로 바꿔주면서 타입선언을 해주어 리터럴 타입으로 변경해주었어요!
그러면 조금 더 타입 safe하게 할 수 있겠죠!